Lusas_Adapter/CRUD/Create/Create.cs

```diff
@@ -374,6 +376,53 @@ private bool CreateCollection(IEnumerable<Edge> edges)
 
         /***************************************************/
 
+        private bool CreateCollection(IEnumerable<Opening> openings)
+        {
+            if (openings != null)
+            {
+                CreateTags(openings);
+
+                List<Opening> validOpenings = new List<Opening>();
+
+                foreach (Opening opening in openings)
+                {
+                    if (CheckPropertyError(opening, p => p.Edges))
+                        if (CheckPropertyError(opening.Edges, e => e.Select(x => x.Curve)))
+                        {
+                            if (opening.Edges.All(x => !Engine.Adapters.Lusas.Query.InvalidEdge(x)))
+                            {
+                                if (Engine.Spatial.Query.IsPlanar(opening, false, Tolerance.MacroDistance)) //Check if this works.
+                                {
+                                        for (int i = 0; i < opening.Edges.Count; i++)
+                                        {
+                                            if (!CheckPropertyError(opening, p => opening.Edges[i]) && Engine.Adapters.Lusas.Query.InvalidEdge(opening.Edges[i]))
+                                                break;
+
+                                            if (i == opening.Edges.Count - 1)
+                                                validOpenings.Add(opening);
+                                        }                               
+                                }
+                                else
+                                    Engine.Base.Compute.RecordWarning("The geometry defining one of the Openings of the Panel is not Planar, and therefore the Opening will not be created.");
```

Comment:
```suggestion
                                    Engine.Base.Compute.RecordError("The geometry defining one of the Openings of the Panel is not Planar, and therefore the Opening will not be created.");
```

If it cannot push the Panel with all it's openings I think this should be an error.